security-utils/src/main/java/com/yahoo/security/tls/TlsContext.java

```diff
@@ -13,6 +13,7 @@
  */
 public interface TlsContext extends AutoCloseable {
 
+    // TODO: Where does this set come from?
```

Comment:
See https://github.com/vespa-engine/vespa/blob/master/vespalib/src/vespa/vespalib/net/tls/impl/iana_cipher_map.cpp#L16-L20

Reply:
Thanks, will resolve TODO in  another PR